csharp/test/Ice/interceptor/AllTests.cs

```diff
@@ -309,6 +309,33 @@ public static IMyObjectPrx Run(TestHelper helper)
                 TestHelper.Assert(request.IsSealed);
             }
             output.WriteLine("ok");
+
+            if (ice2)
```

Comment:
Should we also provide an ice1 test, since these interceptors are not ice2 specific?

Reply:
right I added a simpler ice1 test that doesn't use context override